packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,491 @@
+            'aws:TagKeys': [
+              'BedrockManaged',
+              'BedrockModelCustomizationJobArn',
+            ],
+          },
+        },
+      }),
+    ];
+  }
+
+  private renderPolicyStatements(): iam.PolicyStatement[] {
+    const policyStatements = [
+      new iam.PolicyStatement({
+        actions: [
+          'bedrock:CreateModelCustomizationJob',
+          'bedrock:TagResource',
+        ],
+        resources: [
+          this.props.baseModel.modelArn,
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'custom-model',
+            resourceName: '*',
+          }),
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'model-customization-job',
+            resourceName: '*',
+          }),
+        ],
+      }),
+      new iam.PolicyStatement({
+        actions: ['iam:PassRole'],
+        resources: [this._role.roleArn],
+      }),
+    ];
+    return policyStatements;
+  }
+
+  private validateStringLength(name: string, min: number, max: number, value?: string): void {
+    if (value !== undefined && !Token.isUnresolved(value) && (value.length < min || value.length > max)) {
+      throw new Error(`${name} must be between ${min} and ${max} characters long, got: ${value.length}`);
+    }
+  }
+
+  private validatePattern(name: string, pattern: RegExp, value?: string): void {
+    if (value !== undefined && !Token.isUnresolved(value) && !pattern.test(value)) {
+      throw new Error(`${name} must match the pattern ${pattern.toString()}, got: ${value}`);
+    }
+  }
+
+  private validateArrayLength(name: string, min: number, max: number, value?: any[]): void {
+    if (value !== undefined && (value.length < min || value.length > max)) {
+      throw new Error(`${name} must be between ${min} and ${max} items long, got: ${value.length}`);
+    }
+  }
+
+  /**
+   * Provides the Bedrock CreateModelCustomizationJob service integration task configuration
+   *
+   * @internal
+   */
+  protected _renderTask(): any {
+    return {
+      Resource: integrationResourceArn('bedrock', 'createModelCustomizationJob'),
+      Parameters: sfn.FieldUtils.renderObject({
+        BaseModelIdentifier: this.props.baseModel.modelArn,
+        ClientRequestToken: this.props.clientRequestToken,
+        CustomizationType: this.props.customizationType,
+        CustomModelKmsKeyId: this.props.customModelKmsKey?.keyArn,
+        CustomModelName: this.props.customModelName,
+        CustomModelTags: this.props.customModelTags?.map((tag) => ({ Key: tag.key, Value: tag.value })),
+        HyperParameters: this.props.hyperParameters,
+        JobName: this.props.jobName,
+        JobTags: this.props.jobTags?.map((tag) => ({ Key: tag.key, Value: tag.value })),
+        OutputDataConfig: {
+          S3Uri: this.props.outputData.bucket.s3UrlForObject(this.props.outputData.prefix),
+        },
+        RoleArn: this._role.roleArn,
+        TrainingDataConfig: {
```

Comment:
Just wondering if there's a reason we omit `invocationLogsConfig`? 

https://docs.aws.amazon.com/bedrock/latest/APIReference/API_TrainingDataConfig.html#API_TrainingDataConfig_Contents

Reply:
When it was created, I remember handling all the parameters, so additional arguments might have been added later. Should I address this as well?